bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/loadpoint.xml

```diff
@@ -0,0 +1,614 @@
+			<tag>Switch</tag>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="ON">Activated</option>
+				<option value="OFF">Deactivated</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="loadpointChargeDuration">
+		<item-type>Number:Time</item-type>
+		<label>Charging Duration</label>
+		<description>Time passed since start of the charging</description>
+		<category>Time</category>
+		<state pattern="%.1f min" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargePower">
+		<item-type>Number:Power</item-type>
+		<label>Charging Power</label>
+		<description>Current power of charging</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargeTotalImport">
+		<item-type>Number:Energy</item-type>
+		<label>Charge Total Import</label>
+		<description>Total imported energy</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="loadpointChargeRemainingDuration">
+		<item-type>Number:Time</item-type>
+		<label>Charging Remaining Duration</label>
+		<description>Remaining duration until limit SoC is reached</description>
+		<category>Time</category>
+		<state pattern="%.1f min" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargeRemainingEnergy">
+		<item-type>Number:Energy</item-type>
+		<label>Charging Remaining Energy</label>
+		<description>Remaining energy until limit SoC is reached</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargedEnergy">
+		<item-type>Number:Energy</item-type>
+		<label>Charged Energy</label>
+		<description>Energy charged since plugged-in</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargerFeatureHeating">
+		<item-type>Switch</item-type>
+		<label>Charger Feature: Heating</label>
+		<description>Indicates whether the loadpoint is configured for a heating device</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Status</tag>
+		</tags>
+		<state readOnly="true">
+			<options>
+				<option value="ON">True</option>
+				<option value="OFF">False</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="loadpointChargerFeatureIntegratedDevice">
+		<item-type>Switch</item-type>
+		<label>Charger Feature: Integrated Device</label>
+		<description>Loadpoint has an integrated device and runs without a vehicle (e.g. heatpump, e-bike)</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Status</tag>
```

Comment:
Enabled. Please check all channels for proper semantic tags. 
Possible tags:
https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv

Documentation:
https://next.openhab.org/docs/developer/bindings/semantic-tags.html#developer-guidelines-on-semantic-tags